### ISSUE TITLE
CLI uploaded datasets are still draft

### ISSUE THREAD

**Issue by B-Hartmann** [DCOR-dev/DCOR-Aid#59]
DCOR-Aid version 0.11.0
Windows 10 
Python 3.9.0

I use the CLI to upload data to DCOR, specifically the function `dcoraid.cli.upload_task(path, server api_key)`.
In the command prompt, I can see the following text:
```
Initializing.
Dataset ID is 606523e4-4000-4678-be95-0bb8ff2xxxxx.
Compressing resources.
Uploading resources.
100% (file 2/2)
Verifying upload.
Done.
```
Online on DCOR I can see that all files are uploaded, but the dataset is still marked as a draft and doesn't change into an active state. If it says "Done", I would expect it to be completely done, so it seems that there is no check if the function `dcoraid.api.dataset_activate()` was called successfully.

**Comment by paulmueller** [comment]
fixed in 0.2.4

**Comment by paulmueller** [comment]
This is fixed in 0.11.1. Please create a separate issue with the error message that you (hopefully) get with the new version.

**Comment by paulmueller** [comment]
I think #54 fixes the underlying issue.
